File: exo3.php
<?php

#algoExoQais
#beweb lunel


$result = 0;
$i = 0;
function countLetter($word,$letter)
{
	$result = 0;
	$i = 0;
	$argv[1] = $word;
	$argv[2] = $letter;
	while($i < strlen($word))
	{
		if($word[$i] == $letter)
		{
			$result++;
		}
	$i++;
	}
	echo "le nombre de $letter est : $result\n";
}
countLetter($argv[1], $argv[2])
?>

File: exo3.rb
#algoExoQais
#beweb lunel

$result = 0
$i = 0

def countLetter(word,letter)
	ARGV[0] = $word
	ARGV[1] = $letter
	letter = "t"
	while $i < $word.length-1
		if $word[$i] == $letter
			$result+=1
		end
	$i+=1
	end
	("le nombre de #{$letter} est : #{$result}")
end

countLetter(ARGV[0], ARGV[1])
